spynnaker/pyNN/models/spike_source/spike_source_array.py

```diff
@@ -174,6 +174,10 @@ def set_recording_spikes(self, new_state=True):
         self._requires_mapping = not self._spike_recorder.record
         self._spike_recorder.record = new_state
 
+    @overrides(AbstractSpikeRecordable.get_spikes_sampling_interval)
+    def get_spikes_sampling_interval(self):
+        return globals_variables.get_simulator().machine_time_step
```

Comment:
It may just be that this isn't done yet, but I think this should return ```self._spike_recorder.get_spikes_sampling_interval```

Reply:
I have not yet decided where to keep the spike sample_interval but YES it should be in one place and YES it should be where the parameters are passed to the C doe so that is worth considering.

Reply:
OK - this isn't critical for this pull request in any case

Reply:
Will look at this later when I do selective recording of spikes